### ISSUE TITLE
[java] False positive: method with higher visibility is not UselessOverridingMethod

### ISSUE THREAD

**Issue by StuartClayton5** [pmd/pmd#1689]
**Affects PMD Version:** 
eclipse plugin 4.0.17
**Rule:**
UselessOverridingMethod
**Description:**
An overriding method that increases visibility is not "useless", even when it "merely returns super". PMD flags the method with UselessOverridingMethod. This is a false positive.
**Code Sample demonstrating the issue:**
In class A:
     protected String x() {
         return "abc";
     }
In subclass B of A:
     @Override
     public String x() {
         return super.x();
     }
B,x() is not "useless", this is the only way to make the method available outside of the class package.

```

**Running PMD through:** *[CLI | Ant | Maven | Gradle | Designer | Other]*
Other (eclipse plugin)
<!-- If relevant, also include your JDK and OS information, e.g. for ClassNotFoundException, LinkageError, reflection failures, etc. -->

**Comment by jsotuyod** [comment]
@StuartClayton5 thanks for the report. This seems to be a duplicate of #911, so I'm merging this onto it.
